java/test/jmri/jmrit/beantable/LogixTableActionTest.java

```diff
@@ -34,80 +34,112 @@
     private static LogixTableAction _logixTable;
 
     public void testCreateLogix() throws Exception {
+        if (GraphicsEnvironment.isHeadless()) {
+            return; // cannot use Assume in a TestCase
+        }
         //Setting these two prevents any pop up message generated by the system trying to save messages.
         jmri.InstanceManager.getDefault(jmri.UserPreferencesManager.class).setPreferenceState("jmri.managers.DefaultUserMessagePreferences", "reminder", true);
         jmri.InstanceManager.getDefault(jmri.UserPreferencesManager.class).setPreferenceState("jmri.jmrit.beantable.LogixTableAction", "remindSaveLogix", true);
 
         jmri.InstanceManager.store(new jmri.jmrit.signalling.EntryExitPairs(), jmri.jmrit.signalling.EntryExitPairs.class);
 
-        try {
```

Comment:
The big changes in this file are from removing a try/catch around the tests that merely printed a stack trace and rethrew the exception. In (most) tests, unexpected exceptions get a stack trace printed without this try/catch wrapper, so I removed it.